integration/network/service_test.go

```diff
@@ -314,7 +154,133 @@ func noServices(client client.ServiceAPIClient) func(log poll.LogT) poll.Result
 	}
 }
 
+func TestDaemonRestartWithLiveRestore(t *testing.T) {
```

Comment:
Why were these moved? Could we keep them at the same location to minimise the diff?

Reply:
Moved back.